duckduckgo/Android
# Add tracker blocker for service workers

Task/Issue URL: https://app.asana.com/0/414730916066338/1200317286377247
Tech Design URL: 
CC: 

**Description**:
This PR adds tracker blocker for service workers for API >= 24

Note: This has been tested on APIs < 24to check the app doesn't crash or has any problems.

**Steps to test this PR**:
1. Install from this branch
1. Go to `https://privacy-test-pages.glitch.me/privacy-protections/request-blocking/`
1. Press `Start the test`
1. Scroll down to the `Other` section and check the serviceworker-fetch request is marked as failed (grey square with red border)
1. In the logcat you should also see a few lines similar to `Blocking request https://bad.third-party.site/privacy-protections/request-blocking/block-me/fetch.json?serviceworker-0.12289616434485073`


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
